src/main/java/bookstore/controller/BookController.java

```diff
@@ -30,25 +32,27 @@ public List<BookDto> getAll() {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
 
     @ResponseStatus(HttpStatus.OK)
     @GetMapping("/{id}")
-    public BookDto getBookById(@PathVariable Long id) {
+    public BookDto getBookById(@PathVariable @Min(1) Long id) {
```

Comment:
1. In order to be able to validate such things as `@PathVariable` you need to annotate your controller with `@Validated` annotation
2. it is better to use `@Positive` rather than `@Min(1)`